specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2022-11-15/mongorbac.json

```diff
@@ -406,7 +406,7 @@
           "description": "A user-friendly name for the Role Definition. Must be unique for the database account."
         },
         "type": {
-          "type": "string",
+          "type": "int",
```

Comment:
Maybe it needs additional attribute ""format": "int32"," or ""format": "int64",", right?